test/command_line/test_merge.py

```diff
@@ -98,12 +100,18 @@ def test_merge_dmin_dmax(dials_data, tmpdir):
         "project_name=ham",
         "crystal_name=jam",
         "dataset_name=spam",
+        "best_unit_cell=%s" % best_unit_cell,
     ]
     result = procrunner.run(command, working_directory=tmpdir)
     assert not result.returncode and not result.stderr
 
     # check we only have reflections in range 8 - 1A
```

Comment:
This comment is now separated from the code it refers to - can you move the comment further down?